presto-iceberg/src/test/java/com/facebook/presto/iceberg/hive/TestIcebergHiveStatistics.java

```diff
@@ -402,6 +407,41 @@ public void testPredicateOnlyColumnInStatisticsOutput(boolean pushdownFilterEnab
         }
     }
 
+    @Test
+    public void testStatisticsCachePartialEviction()
+            throws Exception
+    {
+        try (DistributedQueryRunner queryRunner = createIcebergQueryRunner(ImmutableMap.of(), ImmutableMap.of("iceberg.max-statistics-file-cache-size", "1024B"))) {
+            Session session = Session.builder(queryRunner.getDefaultSession())
+                    // set histograms enabled
+                    .setSystemProperty("optimizer_use_histograms", "true")
```

Comment:
nit: should we replace the string with `com.facebook.presto.SystemSessionProperties#OPTIMIZER_USE_HISTOGRAMS`?

Reply:
updated